docs/Translation.md

```diff
@@ -0,0 +1,78 @@
+---
+description: BTCPay Translation features.
+---
+
+# Using BTCPay Translation Feature to Localize Your BTCPay Server instance
+
+
+BTCPay Server now includes a powerful translation feature that allows merchant administrators manage their instance in their preferred language. 
+
+This guide will help you create and manage translations, making the BTCPay platform more user-friendly for you.
+
+With this feature, you can replace the default English text across the entire application with your chosen language. 
+
+Whether you want to customize texts on your checkout page, settings configuration, dashboard, or even error and success messages, BTCPay's translation feature makes it easy to tailor your experience to your linguistic needs.
```

Comment:
Not really useful information, especially it's incorrect, translations currently don't affect checkout. Only the back-office interface.
```suggestion
```